### DISCUSSION TITLE
Can shared admin account be deleted?

### DISCUSSION THREAD

**Discussion by ChiragJaniEdlevo** [DependencyTrack/dependency-track#4126] [Q&A] [answered]
Shared Admin account was setup when we deployed the instance of dependency track. We then have onboarded the more users using OPENID CONNECT. We have now provided admin rights to the required ones; however, we have kept the shared account as well.

I wanted to understand if we removed this admin account from the managed user, will it cause any impact on the application working on future upgrades.

Note: We are using DT version 4.8.2 and will be soon migrating to latest version.

Maybe I have missed or overlooked any documentation in this context.

Looking forward to your support and answers in this regard.

Thanks,
Chirag Jani

**Comment by nscuro** [comment, marked as answer]
We generally recommend to keep it in case the OIDC integration breaks (i.e. your identity provider is down but you *really* need to access the system) or any other emergencies.

Put a very strong password on the admin account and lock the password away somewhere safe. If you don't end up needing it - great. But if you do, you'll be happy you kept the admin account around.

**Reply by ChiragJaniEdlevo** [reply]
Thanks for the information and clarification :) Have a great day!!!
